multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -337,3 +352,25 @@ func (c *Cluster) validateDNSConfig(r53 r53Service) error {
 
 	return nil
 }
+
+func stackEventErrMsgs(events []*cloudformation.StackEvent) []string {
+	var errMsgs []string
+
+	for _, event := range events {
+		if aws.StringValue(event.ResourceStatus) == cloudformation.ResourceStatusCreateFailed {
+			// Only show actual failures, not cancelled dependent resources.
+			if aws.StringValue(event.ResourceStatusReason) != "Resource creation cancelled" {
```

Comment:
That makes sense. I'm fine with ignoring them. And if a user triggers a cancellation, they should likely know why it failed :)